jb/ut_book_depth_statistics.cpp

```diff
@@ -0,0 +1,84 @@
+#include <jb/book_depth_statistics.hpp>
+
+#include <boost/test/unit_test.hpp>
+
+constexpr bool eqTypes() {
+  return std::is_same<jb::itch5::book_depth_t, jb::book_depth_t>::value;}
+
+/**
+ * @test Verify that jb::book_depth_statistics works as expected.
+ */
+BOOST_AUTO_TEST_CASE(book_depth_statistics_simple) {
+  jb::book_depth_statistics::config cfg;
+  jb::book_depth_statistics stats(cfg);
+  
+  stats.sample(1);
+  stats.sample(2);
+  stats.sample(3);
+  stats.sample(4);
+  stats.sample(5);
+
+  BOOST_STATIC_ASSERT_MSG(eqTypes(),
```

Comment:
Oh, I see.  I try to use the standard C++11 features instead of the backwards-compatible macros in Boost.